tests/md-nacl.rs

```diff
@@ -1,7 +1,7 @@
 // Lumol, an extensible molecular simulation engine
 // Copyright (C) Lumol's contributors — BSD license
 
-//! Testing physical properties of a NaCl crystal
+//! Testing physical properties of a sodium chloride crystal
```

Comment:
NaCl trigger a lint (it looks like a type), and give a suggestion to put it inside backticks ...